src/assets/js/protocol/protocol-menu.js

```diff
@@ -297,12 +297,15 @@ if (typeof Mzp === 'undefined') { // eslint-disable-line block-scoped-var
     };
 
     /**
-     * Enables simplified menu using pure CSS hover states.
+     * Enhances the menu for 1st class JS support.
      */
-    Menu.cssFallback = function() {
-        var menu = document.querySelector('.mzp-c-menu');
-        var currentClassName = menu.className;
-        menu.className = currentClassName.replace(/mzp-c-menu/, 'mzp-c-menu mzp-c-menu-basic');
+    Menu.enhanceJS = function() {
+        var menu = document.querySelectorAll('.mzp-c-menu');
+
+        for (var i = 0; i < menu.length; i++) {
+            menu[i].classList.remove('mzp-is-basic');
```

Comment:
Damn. As a team we've been using `.mzp-is-state` and `.is-state` inconsistently.

The menu component already uses `.is-animated` and `.is-selected`.

I opened https://github.com/mozilla/protocol/issues/324 to discuss.

Reply:
Given that the docs suggest using a namespace, I'll update here so the menu component is consistent 👍